src/Tar.ts

```diff
@@ -1,26 +1,20 @@
-import { assertType } from 'typescript-is' 
-import { promises as fs } from 'fs'
-import * as path from 'path'
+import { promises as fs } from "fs";
+import * as path from "path";
 
 export interface Manifest {
-  Config: string
-  RepoTags: string[] | null
-  Layers: string[]
+  Config: string;
+  RepoTags: string[] | null;
+  Layers: string[];
 }
 
-export type Manifests = Manifest[]
-
-export function assertManifests(x: unknown): asserts x is Manifests {
-  assertType<Manifests>(x)
-}
+export type Manifests = Manifest[];
 
 export async function loadRawManifests(rootPath: string) {
-  return (await fs.readFile(path.join(rootPath, `manifest.json`))).toString()
+  return (await fs.readFile(path.join(rootPath, `manifest.json`))).toString();
 }
 
 export async function loadManifests(path: string) {
-  const raw = await loadRawManifests(path)
-  const manifests = JSON.parse(raw.toString())
-  assertManifests(manifests)
-  return manifests
+  const raw = await loadRawManifests(path);
+  const manifests = JSON.parse(raw.toString()) satisfies Manifests;
```

Comment:
I see, and looks like the builds are happy with it too.  This work is satisfactory :laughing: